test/MerklePatriciaProof.js

```diff
@@ -0,0 +1,115 @@
+const MerklePatriciaProofMock = artifacts.require("./MerklePatriciaProofMock.sol");
+const proofJson =  require("./data/proof.json");
+const ethUtil = require ("ethereumjs-util");
+contract('MerklePatriciaProof', function(accounts) {
+
+    let merkleMock= null;
+
+    describe('Test Cases for Account proof verification', async () => {
+        before(async () => {
+            merkleMock = await MerklePatriciaProofMock.new();
+        })
+
+        it('Should pass', async () => {
+            let encodedValue = '0x'+ethUtil.sha3(proofJson.account.rlpEncodedAccount).toString('hex');
```

Comment:
As the data used for testing is static, we can save encoded value in proof.json itself.

Reply:
Added an new key in proof.json.